users/README.md

```diff
@@ -2,43 +2,48 @@
 
 ```json
 {
-	"id": "string",
-	"username": "string",
-	"first_name": "string",
-	"last_name": "string",
-	"email": "string",
-	"phone": "number",
-	"yoe": "number",
-	"company": "string",
-	"designation": "string",
-	"img": "string",
-	"github_id": "string",
-	"github_display_name": "string",
-	"linkedin_id": "string",
-	"twitter_id": "string",
-	"instagram_id": "string",
-	"skills": [],
-	"joined_discord": "string",
-	"website": "string",
-	"isMember": "boolean",
-	"userType": "string",
-	"tokens": {
-		"githubAccessToken": "string"
-	},
-	"status": "string",
-	"roles": {
-		"app_owner": "boolean",
-		"archived": "boolean",
-		"member": "boolean",
-		"restricted": "boolean",
-		"super_user": "boolean",
-		"in_discord": "boolean"
-	},
-	"profileURL": "string",
-	"picture": {
-		"publicId": "string",
-		"url": "string"
-	}
+  "id": "string",
+  "username": "string",
+  "first_name": "string",
+  "last_name": "string",
+  "email": "string",
+  "phone": "number",
+  "yoe": "number",
+  "company": "string",
+  "designation": "string",
+  "img": "string",
+  "github_id": "string",
+  "github_display_name": "string",
+  "linkedin_id": "string",
+  "twitter_id": "string",
+  "instagram_id": "string",
+  "skills": [],
+  "joined_discord": "string",
+  "website": "string",
+  "isMember": "boolean",
+  "userType": "string",
+  "tokens": {
+    "githubAccessToken": "string"
+  },
+  "status": "string",
+  "roles": {
+    "app_owner": "boolean",
+    "archived": "boolean",
+    "member": "boolean",
+    "restricted": "boolean",
+    "super_user": "boolean",
+    "in_discord": "boolean"
+  },
+  "archivedDetails": {
```

Comment:
How many archived details are we saving, only one or more? If more than one why is it not an array?

Reply:
Only one for one user